### ISSUE TITLE
This might not be safe.  Can someone confirm this code is for being able to take ownership of my NFTs and weth?

### ISSUE THREAD

**Issue by Negaunee0024** [nftdevs/NFTs-Upload-to-OpenSea#34]
I used this a few days ago at night time, I figured I would leave it on overnight.  I had security issues on my PC in the morning and couldn't use my pin to log in, I had to reinstall windows.  I just checked tho and there is an allowance on my wallet to spend unlimited weth and a weird smart contract.  I created this wallet from before downloading this as I had concerns.  The only transaction I approve was to initialize my wallet on opensea.  Heres my wallet;  
0xd8e833d9563f0795c8619a3843e75a9957169a83
Here's said smart contract that I can't revoke until I created a username on etherscan which I will do soon. 
https://etherscan.io/address/0xe5c783ee536cf5e63e792988335c4255169be4e1#code

**Comment by Negaunee0024** [comment]
![image](https://user-images.githubusercontent.com/96413524/146737735-321add47-769f-49ad-99dd-0718b22efaa5.png)

**Comment by Negaunee0024** [comment]
![image](https://user-images.githubusercontent.com/96413524/146738132-3a927690-f96d-4062-a232-7df07f352884.png)

**Comment by Asthetic** [comment]
@Negaunee0024 Well, the chrome executable is some arbitrary executable. It's possible it made changes to your wallet. I don't believe I had to sync my wallet or anything, I just downloaded my own chromedriver for Selenium. 

The code itself looks clean, but that EXE could have done it. /shrug

**Comment by Negaunee0024** [comment]
Thanks.

**Comment by Negaunee0024** [comment]
I feel dumb even following a guide on  how to do this without having a coding background.  It's like I'm setting myself up for disaster.  I was embarrassed to write this post even, but something happened.  This was a fresh wallet created just for this, so it is related for surely.  Time to study up more on coding.

**Comment by Asthetic** [comment]
> I feel dumb even following a guide on how to do this without having a coding background. It's like I'm setting myself up for disaster. I was embarrassed to write this post even, but something happened. This was a fresh wallet created just for this, so it is related for surely. Time to study up more on coding.

Well, not really. I wouldn't have spent time combing through the code if it weren't for this post. I'm looking into some of the dependencies because I don't know what they are and those could potentially be malicious as well. 

What it comes down to is this: this code runs that executable file and we don't really know what it does. 

I have coding experience, but not a ton with Python. I might just use this code's Selenium actions and write my own code to make sure it's safe. Plus, this code doesn't deal with attributes at all which I want for my NFT.

**Comment by Asthetic** [comment]
I just spent some time looking at dependencies, they look fine to me. It's also possible that what you posted is some default contract that was added by uploading an NFT and publishing it. I'd try Googling the contract name.

**Comment by Negaunee0024** [comment]
Yeah I also spent several hours researching to see if anything stuck out in the smart contract.  Opensea was the only place I used the wallet at.  Wyvern Protocol v2 Mainnet Deployment is the name of the smart contract, and on their etherscan it is just failed transactions (and one successful from 20 days ago)  Extremely strange.  Thanks for taking the time to check, I def appreciate it.  Here's their etherscan with their contract address and all the failed transactions, https://etherscan.io/address/0xe5c783ee536cf5e63e792988335c4255169be4e1       
I also found the "Wyvern protocol v2 mainnet github"  https://github.com/ProjectWyvern/wyvern-ethereum
-Seems strange that Opensea would use such a feature, as only seems to have 1 or 2 transactions a day posted and are mainly failed.  Appreciate your time for sure.

**Comment by Negaunee0024** [comment]
Yeah the attributes are a big thing for me too.  I just wanted to test this out as an option.  I was having an issue with the NFT's not posting (Metamask transactions not being signed)  so I erased some code and it uploaded over 4,200 overnight for me (none for sale tho, since that feature didnt' work)  This github looks promising so far, it's an entire 721 deployment with attributes and all-def way more involved, but looks like it will be worth it.  I just started checking it out an hour or so ago  https://github.com/PatrickAlphaC/nft-mix

**Comment by Extramax** [comment]
Now I've got why he always says that he doesn't like Mini Mouse Macro.. Because in this script there's something that brings him money.

**Comment by Moismailov** [comment]
I got locked out of my account on windows after using this can't even log back in... also saw a .DAT extension file that disappeared from my desktop before this happened

**Comment by negaunee042** [comment]
> I got locked out of my account on windows after using this can't even log back in... also saw a .DAT extension file that disappeared from my desktop before this happened

Same thing happened to me.  That was the first thing that got my attention.  I was able to fix it by changing some things in safe mode I believe.  I deleted all of this software.  Thankfully I have a cold storage wallet.  Just this week someone tried to port-out my phone number.  Not sure if it's related, but I would delete this immediately.  Not worth it at all.  I decided to learn solidity to deploy smart contracts so I don't have to do things like this ever again.
